standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -2255,7 +2259,7 @@ public Table getTable(String catName, String dbName, String tableName,
       if (processorIdentifier != null)
         req.setProcessorIdentifier(processorIdentifier);
 
-      Table t = client.get_table_req(req).getTable();
+      Table t = getTableFromLocalCache(req);
```

Comment:
Same comment as above for `getTableFromLocalCache`.

Reply:
And, I'll rename it to `getTable`.